lib/driverProviders/sauce.ts

```diff
@@ -63,10 +63,10 @@ export class Sauce extends DriverProvider {
     this.config_.capabilities['username'] = this.config_.sauceUser;
     this.config_.capabilities['accessKey'] = this.config_.sauceKey;
     this.config_.capabilities['build'] = this.config_.sauceBuild;
-    let auth = 'https://' + this.config_.sauceUser + ':' + this.config_.sauceKey + '@';
-    this.config_.seleniumAddress =
-        auth + (this.config_.sauceSeleniumAddress ? this.config_.sauceSeleniumAddress :
-                                                    'ondemand.saucelabs.com:443/wd/hub');
+    let auth = this.config_.sauceUser + ':' + this.config_.sauceKey + '@';
+    this.config_.seleniumAddress = this.config_.sauceSeleniumAddress ?
+        ('http://' + auth + this.config_.sauceSeleniumAddress) :
```

Comment:
@ndmanvar i just noticed that we also need this fix to support protractor 5.1.1+ as the new forcing to https for ondemand does not work through our inspection proxy.   We would need to set sauceSeleniumAddress to ondemand.saucelabs.com:80/wd/hub to work around this.. (unless we make the scheme another parameter as mentioned previously..)  - if you can at least make the style fix suggested by @cnishina that would be great.